pages/common/git-blame.md

```diff
@@ -5,11 +5,11 @@
 
```

Comment:
Warning: page is not in `pages.ta/` directory: PR name doesn't reflect this fact.

Reply:
> Warning: page is not in `pages.ta/` directory: PR name doesn't reflect this fact.

Sorry, I forgot to add it to the PR description, will update it. Since this is a minor change, I don't think we need to include it in title.